roq-theme/default/src/main/web/roq.scss

```diff
@@ -150,6 +144,7 @@ pre {
 }
 
 .post, .article-page, .page-content  {
+	width: 100%;
```

Comment:
Looks like one minor spacing issue here but other than that looks good!

Reply:
fixed